panasonic_camera/camera.py

```diff
@@ -157,6 +162,14 @@ def get_info_capability(self) -> Capability:
             states=find_all_text(camrply, 'getstatelist/getstate'),
             specifications=find_all_text(camrply, 'camspeclist/camspec'))
 
+    def register_with_camera(self, device_name='robot-cameraman'):
+        # Cameras like the DC-FZ80 keep a list of devices that remote
+        # control them. This request adds the current device to the list with
+        # a name specified by device_name.
+        return self._request(
+            params={'mode': 'accctrl', 'type': 'req_acc', 'value': '0', 'value2': device_name}
+        )
```

Comment:
Line is too long
```
PEP 8: E501 line too long (94 > 80 characters)
```
I'd prefer

```python
return self._request(
    params={
        'mode': 'accctrl',
        'type': 'req_acc',
        'value': '0',
        'value2': device_name
    })
```

Reply:
Fixed in 3cf95c992d09b4a34145fc04fc7c32c9871f894f